azure-pipelines/templates/steps/maven-release/gpg-signing.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
```

Comment:
I believe this was earlier being done as an inline script? I like that you've moved it to its own file 👍

Reply:
We probably want to add license to these script files

Reply:
Yes it is a inline scripts that is very similar for the different products that I just adapted into one